src/lib/Controller/api/display.c

```diff
@@ -761,8 +763,8 @@ WbImageRef wb_display_image_load(WbDeviceTag tag, const char *filename) {
     return NULL;
   }
   GImage *gi = g_image_new(filename);
-  if (gi->failed || (gi->data_format != G_IMAGE_DATA_FORMAT_ABGR && gi->data_format != G_IMAGE_DATA_FORMAT_RGB)) {
-    fprintf(stderr, "Error: wb_display_image_load(): the \"%s\" image is unreadable.\n", filename);
```

Comment:
Why did you remove this error message?

Reply:
Oups that's a mistake, I will restore it.